schemas/paconn-apiProperties.schema.json

```diff
@@ -434,33 +429,15 @@
             "customParameters": {
               "additionalProperties": false,
```

Comment:
When we have additionalParameters, should they be allowed in customParameters? Please check NetDocuments config if it is not correct or if we need to have "additionalProperties": true

Reply:
oh right. that's true. The additionalParameters are only usable when the identityProvider has some '*Template' customParameters defined. I'll remove this restriction for the known identityProviders that do have Template parameters.